src/sas/sasgui/perspectives/fitting/fitting.py

```diff
@@ -1788,8 +1794,13 @@ def _complete1D(self, x, y, page_id, elapsed, index, model,
                     self._plot_residuals(page_id=page_id, data=data, fid=fid,
                                          index=index, weight=weight)
 
-            msg = "Computation  completed!"
+            msg = "Computation completed!"
+            if number_finite != len(y):
```

Comment:
if `y` is not 1 dimensional (perhaps not out of bounds of possibility) then this statement will be True.
Use `if number_finite != y.size`